src/siembol/Chart.yaml

```diff
@@ -0,0 +1,55 @@
+# Chart for siembol
+apiVersion: v2
+description: siembol is a security analytics framework akin to projects like Apache Metron
+name: siembol
+type: application
+version: 0.1.0
+#appVersion: 0.0.1
+keywords:
+- kafka
+- zookeeper
+- storm
+- gitlab
+- hdfs
+- security
+- metron
+home: https://github.com/G-Research/charts
+sources:
+- https://github.com/G-Research/siembol
+- https://github.com/G-Research/charts/siembol
+maintainers:
+- name: stackedsax
+  email: alex.scammon@ext.gresearch.co.uk
+- name: generalfuzz
+  email: headphonejames@gmail.com
+#icon: https://kafka.apache.org/images/logo.png
+
+dependencies:
+- name: storm
+  version: ~1.0.2
+  repository: https://g-research.github.io/charts
+- name: kafka
+  version: ~0.20.9
+  repository: http://storage.googleapis.com/kubernetes-charts-incubator
+- name: hdfs
+  version: ~0.1.4
```

Comment:
can we use version 3.1.1 ?

Reply:
Version 0.1.4 of the hdfs chart will run hdfs 2.7.7.  Is this recent enough?

You can see the appVersion the Chart we're using is using here:
* https://github.com/Gradiant/charts/blob/master/charts/hdfs/Chart.yaml#L2

Reply:
We can try and see but the current code is running on hadoop 3.1.1

Reply:
Roger that.  What's interesting is that even the hadoop chart in the stable repo for helm is only using 2.9.0:

* https://github.com/helm/charts/blob/master/stable/hadoop/Chart.yaml#L5

We'll experiment with bumping the version and crossing our fingers...